packages/iris-grid/src/sidebar/RollupRows.tsx

```diff
@@ -440,22 +442,26 @@ class RollupRows extends Component<RollupRowsProps, RollupRowsState> {
     isClone?: boolean;
     selectedCount?: number;
   }): ReactElement {
-    const indent = isClone ? '' : '\u00A0\u00A0'.repeat(itemIndex);
+    const indent =
+      isClone !== undefined && isClone ? '' : '\u00A0\u00A0'.repeat(itemIndex);
     const text = `${indent}${item}`;
-    const badgeText = isClone ? `${selectedCount}` : undefined;
-    const className = isClone ? 'item-list-item-clone' : '';
+    const badgeText =
+      isClone !== undefined && isClone ? `${selectedCount}` : undefined;
+    const className =
+      isClone !== undefined && isClone ? 'item-list-item-clone' : '';
     return (
       <>
         {DraggableItemList.renderTextItem({ text, badgeText, className })}
-        {!isClone && (
-          <Button
-            kind="ghost"
-            className="btn btn-link btn-link-icon btn-delete-grouping float-right"
-            onClick={() => this.handleDeleteClicked(itemIndex)}
-          >
-            <FontAwesomeIcon icon={vsTrash} />
-          </Button>
-        )}
+        {isClone === undefined ||
+          (!isClone && (
```

Comment:
Right now the case for `isClone === undefined` will no longer render anything. Should do `&&` instead of `||`